src/Sit.scala

```diff
@@ -89,7 +90,25 @@ class Sit(private val projectPath: String,
    * print out the full commit list
    */
   def history(): Unit = {
-    // TODO
+    // TODO replace stub with real commit
+    val stub = Commit(
+      randomUUID().toString,
+      None,
+      Diff(Map("a" -> "b", "c" -> "d"), Map()),
+      "initial commit",
+      0)
+    println(s"Commit: ${stub.id} ")
+    println(s"    ${stub.commitMessage} ")
+    println(s"TimeStamp: ${stub.timestamp}")
+    stub.diff.added.foreach(pair => {
+      println((s"+ ${pair._1} = ${pair._2}"))
+    })
+    stub.diff.deleted.foreach(pair => {
+      println((s"- ${pair._1} = ${pair._2}"))
+      })
+    if (stub.parent != None) {

```

Comment:
One good thing of Scala.util.Option is that you don't have to brutally handle null case : P
`stub.parent.isEmpty()`

It contains lots of handy helper methods, try looking up the documentation of Scala: 
https://www.scala-lang.org/api/current/scala/Option.html